www/artists/jnugent/projects.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            John Nugent
+        </title>
+        <link rel="stylesheet" href="style.css" type="text/css">
+        <style type="text/css">
+        div.c1 {text-align: center}
+        </style>
+    </head>
+    <body>
+        <div id="wrapper">
+            <div id="header">
+                <div class="c1">
+                    <img src="header.jpg" width="800" height="275" alt="no alt text">
+                </div>
+            </div>
+            <div id="navigation">
+                <a href="index.html">Home</a> <a href="projects.html">Projects</a> <a href="../">Saskartists</a>
+            </div>
+            <div id="content">
+                <img src="scuplture_garden1.jpg" alt="no alt text">
+
+                <img src="scuplture_garden2.jpg" alt="no alt text">
+
+                <img src="scuplture_garden3.jpg" alt="no alt text">
+
+                <img src="scuplture_garden4.jpg" alt="no alt text">
+                 <p>Pictures from John's sculpture garden, 2010.</p>
+                <img src="scuplture_garden5.jpg" alt="no alt text">
+                 <p>untitled - snow drifts - 2005 - digital photography</p>
+            </div>
+            <div id="ex4">
+                <div class="c1">
+                    <img src="john_th.jpg" alt="no alt text">
+                </div>
+                <p>
+                    Born January 5, 1921 in Montreal, Quebec. Moved to Lumsden, Saskatchewan in 1941.
+                </p>
+            </div>
+            <div id="footer">
+                <p>
+                    Created in 2012 by Keone Rohatensky
```

Comment:
I'm assuming you remade this page? If so, you should update the footer to reflect this.